Source/ASDisplayNode+Beta.h

```diff
@@ -199,6 +199,7 @@ AS_EXTERN void ASDisplayNodePerformBlockOnEveryYogaChild(ASDisplayNode * _Nullab
 
 @property BOOL yogaLayoutInProgress;
 @property (nullable, nonatomic) ASLayout *yogaCalculatedLayout;
+@property (weak, readonly) ASDisplayNode *yogaRoot;
```

Comment:
I might declare this a method, rather than an atomic weak property. Just to be more transparent.